src/_general.scss

```diff
@@ -54,4 +60,49 @@ a{
         box-shadow: 0 0 10px $brand-color;
         cursor: pointer;
     }
+}
+
+.cancel-button{
```

Comment:
Wouldn't it be much faster to use the buttons that bootstrap already provides?

Reply:
Yes and no because we need to customize them to Derbit's color palet...

Reply:
I think for now it's fine to use the bootstrap standard and then at some point use our own color scheme.